app/controllers/qa_controller.rb

```diff
@@ -29,7 +29,7 @@ def facets
   def page
     params.permit(:page)
     params[:page] = 1 if params[:page].nil?
-    params[:page].to_i.clamp(1, facets.length)
+    params[:page].to_i.clamp(1, facets.length + 1)
```

Comment:
Is `params[:page] = 1 if params[:page].nil?` needed? Even if `params[:page]` is `nil` the next line will return `1`:
```
[1] pry(main)> nil.to_i.clamp(1,5)
=> 1
```

Reply:
nice one, will remove